app/controllers/application_controller/ci_processing.rb

```diff
@@ -272,6 +274,10 @@ def get_rec_cls
     end
   end
 
+  def display_vm_template?
```

Comment:
Why is this method necessary, is it being used elsewhere?

Reply:
For now not, but I can see possibilities to be used in more places.